app-cdr/burn-cd/burn-cd-1.8.1-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-single-r1
+
+DESCRIPTION="Smart console frontend for the cdrkit/cdrtools & dvd+rw-tools"
+HOMEPAGE="https://github.com/aglyzov/burn-cd/"
+SRC_URI="https://github.com/aglyzov/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	virtual/cdrtools
+	app-cdr/dvd+rw-tools"
+
+src_install() {
+	newbin ${P} ${PN}
+	insinto /etc
+	newins dotburn-cd.conf burn-cd.conf
+}
```

Comment:
You should fix shebang on `${PN}` even though it currently only support python2. 

Also add `einstalldocs` here somewhere.